apps/prairielearn/src/server.ts

```diff
@@ -2061,8 +2067,9 @@ export async function startServer(app: express.Express) {
   });
   server.on('connection', () => connectionCounter.add(1));
 
-  // Hack to get us running in Bun, which doesn't currently support `getConnections`:
+  // @ts-expect-error: Hack to get us running in Bun, which doesn't currently support `getConnections`:
   // https://github.com/oven-sh/bun/issues/4459
+  // eslint-disable-next-line @typescript-eslint/no-unnecessary-condition
```

Comment:
Aside, looks like they have an agent working on this: https://github.com/oven-sh/bun/pull/22275